### PR TITLE
standby + auto-wakeup example

### PR DESCRIPTION
**Pull request by recallmenot**
Marek M did the groundwork, I spent the day to get it to work.
Hope this is good enough to be pulled.

### CODE REVIEW

File: examples/standby/standby.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
+// based on https://paste.sr.ht/blob/b9b4fb45cbc70f2db7e31a77a6ef7dd2a7f220fb
+// Could be defined here, or in the processor defines.
+#define SYSTEM_CORE_CLOCK 48000000
+
+#include "../../ch32v003fun/ch32v003fun.h"
```
Comment (AlexanderMandera) [inline examples/standby/standby.c:5]:
```suggestion
#include "ch32v003fun.h"
```

File: examples/standby/standby.c [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+	RCC->APB2PCENR |= RCC_APB2Periph_GPIOD;
+	// GPIO D4 Push-Pull
+	GPIOD->CFGLR &= ~(0xf<<(4*4));
+	GPIOD->CFGLR |= (GPIO_Speed_10MHz | GPIO_CNF_OUT_PP)<<(4*4);
+	GPIOD->OUTDR |= (1 << 4);
+
+	// give the user time to open the terminal connection
+	//Delay_Ms(5000);
+	printf("5000ms wait over\r\n");
```
Comment (AlexanderMandera) [inline examples/standby/standby.c:32]: Should this be here, when the delay is commented out?

**Review by AlexanderMandera — CHANGES_REQUESTED** [review]

File: examples/standby/standby.c (hunk 1 above)
Comment (recallmenot) [reply examples/standby/standby.c:5]: my LSP (clangd) doesn't read the include paths in the makefiles, hence the relative path.
Besides being ugly, is it bad?

Comment (cnlohr) [reply examples/standby/standby.c:5]: The ugliness and inconsistency with the rest of the project is rough.  Please find a way of making your dev environment include that include path.

**Comment by cnlohr** [comment]
It's this ready for merging?

**Pull request merged**